### PR TITLE
Sd boot fixes

### PR DESCRIPTION
**Pull request by aanisov**
Start DomD from mmcblk0p2 and change partitions layout for android.

### CODE REVIEW

**Comment by aanisov** [comment]
updated with patch for domF

**Review by andr2000 — APPROVED** [review]

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg [hunk 1]
```diff
@@ -8,7 +8,7 @@ name = "DomF"
 
 kernel = "/xt/domf/Image"
 
-extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domf ip=192.168.1.13 rw rootwait console=hvc0"
+extra = "root=/dev/mmcblk0p3 rw rootwait console=hvc0"
```
Comment (andr2000) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg]: don't we need ip=dhcp here?

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-salvator-x-h3.cfg [hunk 2]
```diff
@@ -13,7 +13,8 @@ kernel = "/xt/domd/Image"
 device_tree = "/xt/domd/domd.dtb"
 
 # Kernel command line options
-extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd ip=192.168.1.11 rw rootwait console=hvc0 cma=128M pvrsrvkm.DriverMode=0"
+#extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd ip=192.168.1.11 rw rootwait console=hvc0 cma=128M pvrsrvkm.DriverMode=0"
+extra = "root=/dev/mmcblk1p2 rw rootwait console=hvc0 cma=128M pvrsrvkm.DriverMode=0"
```
Comment (andr2000) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-salvator-x-h3.cfg]: ip=dhcp?

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg (hunk 1 above)
Comment (aanisov) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg]: Do we really need dhcp at the kernel boot time? Don't we have a dhcp client in DomF?

Comment (andr2000) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg]: donno

**Comment by aanisov** [comment]
Added patch to provide more resources for DomD.

**Comment by aanisov** [comment]
updated with rb's

**Pull request merged**